docs/src/main/paradox/discovery/index.md

```diff
@@ -482,6 +482,46 @@ setting `akka.discovery.aws-api-ec2-tag-based.tag-key` to something else.
   but will _discover_ the "host" IP (the EC2 private IP) on the AWS API. See @ref:[Basic
   Configuration](../akka-management.md) on how to separate the bind IP from the discovery IP.
 
+* Configuration of the EC2 client is supported and can be done via `application.conf`, by setting
+`akka.discovery.aws-api-ec2-tag-based.client-config` to the fully qualified class name of a class that extends
+`com.amazonaws.ClientConfiguration`.
+
+    Scala
```

Comment:
Formatting seems to be broken (at least when GitHub renders it anyway):
![screenshot 2018-12-08 at 14 02 19](https://user-images.githubusercontent.com/6965859/49686805-e58bb200-faf1-11e8-89e8-fd1d99ab6b25.png)

Reply:
@dhpiggott Fixed; Also, I changed it to use the snip directive :-)  Here's how it looks when paradox renders it:  
![image](https://user-images.githubusercontent.com/5114916/49694943-c28bed00-fb47-11e8-9b55-004efa5ee830.png)